### ISSUE TITLE
HF4 Map is outdated.

### ISSUE THREAD

**Issue by Demiko** [nornagon/hf-mission-planner#21]
The HF4 map in the mission planner is not the same as the map from official TTS mod. The most obvious difference is the node structure around Venus.

![HighFrontier4-map](https://user-images.githubusercontent.com/1906483/87790791-d4686280-c849-11ea-9285-97c284f39aad.jpg)

**Comment by nornagon** [comment]
Thanks, I pulled the version I have at a time before the final version was released. I'll grab the updated version.

**Comment by nornagon** [comment]
I updated using the version from the Vassal module (https://drive.google.com/drive/folders/1JkF9eKRJ89MxdUEYI7da7Cw1uT6C6uja?ths=true) since that had a slightly better quality map image. It took a while because I had to reposition every node by hand 😅 not sure if that's because of a slightly different aspect ratio, or because the nodes themselves were shifted on the board. Anyway, it's done now. Enjoy!
